config/kernelci.conf

```diff
@@ -19,7 +19,7 @@ ssh_host: 172.17.0.1
 output: /home/kernelci/output
 
 [fstests_runner]
-output: /home/kernelci/alex/fstests-poc/output
+output: /home/alexandra/fstests-poc/output
```

Comment:
This gets undone in the next commit, so please just drop it.  It probably was added here by mistake from a local change on your laptop.

Reply:
Still not resolved...  It's a trivial thing and it's really adding unnecessary noise in the git diff, which can lead to unnecessary conflicts when rebasing etc.